test/router.test.ts

```diff
@@ -7,7 +6,7 @@ describe('router', (): void => {
 
   Object.keys(Routes).forEach((name: string): void => {
     it(`has a route named ${name}`, (): void => {
-      expect(router.route(Routes[name])).toBeInstanceOf(Layer);
+      expect(router.route(Routes[name])).toBeTruthy();
```

Comment:
No way to access the object unfortunately. `route()` returns a `Layer` or `false`.

Reply:
to be an object (stronger than being truthy)?

Reply:
(no idea of the matchers available here)

Reply:
https://github.com/libero/article-store/pull/165/commits/92f126a3363e636917734b11238e2d190cca4969 is the best I can think of that still gives a useful message.